android/project.json

```diff
@@ -16,7 +16,6 @@
     ],
     "excludes": [
         ".*",
-        "node_modules",
```

Comment:
Why are the node_modules removed from all the excludes? These get put there when you do npm install and the loctool should not look at anything in the node_modules dirs...

Reply:
node_modules/ is default excluded directory. so it doesn't need to be added additionally. This feature was added in 2.11.0
https://github.com/iLib-js/loctool/blob/development/loctool.js#L174  but so it doesn't work properly because there is a bug. but it's fixed now. so I update the config file here
https://github.com/iLib-js/loctool/pull/175